lib/grp.gi

```diff
@@ -221,13 +223,13 @@ BindGlobal( "IS_PGROUP_FROM_SIZE",
       return true;
     elif s = infinity then
       return false;
-    elif not IsPrimePowerInt( s ) then
+    fi;
+    s := SmallestRootInt( s );
```

Comment:
It really is a nitpick, but since you renamed s in the previous function to p, could you do it here, as well? For better readability?